app/controllers/uploads_controller.rb

```diff
@@ -139,8 +138,14 @@ def validation_messages
         generic_requirement_message('These columns can only contain numeric values: ', validations)
       when ActiveRecord::Validations::UniquenessValidator
         generic_requirement_message('These columns should contain unique values: ', validations)
+      when "#{klass.name}Validator".safe_constantize
+        "#{klass.name}Validator::REQUIREMENT_DESCRIPTIONS".safe_constantize
+      else
+        validations.class.to_s
```

Comment:
Why?  Do you want to just print the name of a validator class to the screen?

Reply:
Adding default behavior, if removing this might have to add a `.select(&:present)` to messages in `requirements_messages` to remove blank lines

Reply:
Isn't having to `.select(&:present)` better than ending up with a bullet point in the view that says something like 
* ActiveRecord::Validations::WhateverValidator

That's what you'll get if the `else` path occurs.

Reply:
removed